CMakeLists.txt

```diff
@@ -285,5 +291,7 @@ message("             boost system Library: ${DATAFED_BOOST_SYSTEM_LIBRARY_PATH}
 message("         boost filesystem Library: ${DATAFED_BOOST_FILESYSTEM_LIBRARY_PATH}")
 message("    boost program_options Library: ${DATAFED_BOOST_PROGRAM_OPTIONS_LIBRARY_PATH}")
 message("                     zlib Library: ${DATAFED_ZLIB_LIBRARIES}")
+message("                   gssapi Library: ${DATAFED_GSSAPI_LIBRARIES}")
```

Comment:
What happens if `BUILD_AUTHZ_TESTS` is false and these are not set?

Reply:
They are just printed as empty. 

zlib Library: 

with nothing after wards.